scripts/bulk_update_leads_info.py

```diff
@@ -56,12 +56,15 @@ def get_contact_info(contact_no, csv_row, what, contact_type):
     * opportunity[0-9]_status           - opportunity status
     * opportunity[0-9]_date_won         - opportunity date won
 
-contact columns (new contacts wil be added):
+contact columns (new contacts will be added):
     * contact[0-9]_name                 - contact name
     * contact[0-9]_title                - contact title
     * contact[0-9]_phone[0-9]           - contact phones
     * contact[0-9]_email[0-9]           - contact emails
     * contact[0-9]_url[0-9]             - contact urls
+
+custom fields (new custom types will be created, new fields will be added):
```

Comment:
this part

https://github.com/congocongo/closeio-api/blob/master/scripts/bulk_update_leads_info.py#L97-L111

2015-04-01 1:27 GMT+07:00 Phil Freo <notifications@github.com>:

> In scripts/bulk_update_leads_info.py
> <https://github.com/elasticsales/closeio-api/pull/39#discussion_r27507945>
> :
>
> >      * contact[0-9]_name                 - contact name
> >      * contact[0-9]_title                - contact title
> >      * contact[0-9]_phone[0-9]           - contact phones
> >      * contact[0-9]_email[0-9]           - contact emails
> >      * contact[0-9]_url[0-9]             - contact urls
> > +
> > +custom fields (new custom types will be created, new fields will be added):
>
> what do you mean by "new custom types will be created"?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/elasticsales/closeio-api/pull/39/files#r27507945>.
>